### DISCUSSION TITLE
[功能请求]: 大佬，求个DOCKER部署……

### DISCUSSION THREAD

**Discussion by themis0009** [GaiZhenbiao/ChuanhuChatGPT#620] [Q&A]
### 相关问题

萌新求DOCKER的参数，连要映射哪个端口都没看明白=  =

### 可能的解决办法

求Docker的环境参数……environment那个~_~谢谢

### 帮助开发

- [ ] 我愿意协助开发！

**Comment by kkkk24juastin** [comment]
docker run -d --name chatgpt \
	-e my_api_key="sk------" \
	-v ~/chatGPThistory:/app/history \
	-v ~/ChuanhuChatbot.py:/app/ChuanhuChatbot.py \
	-p 7860:7860 \
	tuchuanhuhuhu/chuanhuchatgpt:latest


这是我的，映射了历史记录和配置文件

**Comment by themis0009** [comment]
> docker run -d --name chatgpt -e my_api_key="sk------" -v ~/chatGPThistory:/app/history -v ~/ChuanhuChatbot.py:/app/ChuanhuChatbot.py -p 7860:7860 tuchuanhuhuhu/chuanhuchatgpt:latest
> 
> 这是我的，映射了历史记录和配置文件

感谢大佬~
请教，国内服务器部署的话，可以添加反代么？

**Comment by kkkk24juastin** [comment]
你指nginx反代还是openai的api反代，前者的话，可以、后者不清楚，因为我的服务器在海外，不过应该可以更换api的endpoint
、

**Comment by themis0009** [comment]
> 你指nginx反代还是openai的api反代，前者的话，可以、后者不清楚，因为我的服务器在海外，不过应该可以更换api的endpoint 、

谢谢大佬~是指openai的反代……
要是大佬能为我指点nginx的反代就更感谢了

**Comment by kkkk24juastin** [comment]
api反代应该是加上这个把    -e api_host="替换成自定义的api请求地址"、后面的地址是你自己搭建的或者用的别人的，nginx的反向代理，如果是新手的话，建议使用nginxproxymanager的docker镜像或者安装1panel运维面板，操作非常简单

**Reply by themis0009** [reply]
> api反代应该是加上这个把 -e api_host="替换成自定义的api请求地址"、后面的地址是你自己搭建的或者用的别人的，nginx的反向代理，如果是新手的话，建议使用nginxproxymanager的docker镜像或者安装1panel运维面板，操作非常简单

明白了，感谢~
